docs/examples/index.md

```diff
@@ -7,6 +7,7 @@ scale well due to lock contention.
 - [Mandelbrot Set Visualization](mandelbrot.md)
 - [Monte Carlo Simulation](monte-carlo.md)
 - [Web Scraping with asyncio](asyncio.md)
+- [Using `AtomicDict.reduce()` for Multithreaded Aggregation](https://dpdani.github.io/cereggii/examples/AtomicDict/reduce/)
```

Comment:
I think we should break this out into another listing of examples that are outside the guide.

We could also link to the [nvidia blog post about multithreaded data loading](https://developer.nvidia.com/blog/improved-data-loading-with-threads/).